src/sp/it/pl/util/system/Environment.kt

```diff
@@ -95,18 +95,18 @@ fun runCommand(command: String, then: Consumer<Process>? = null) {
 
 /**
  * Browse the file in OS' file browser:
- * * if denotes a directory it will be opened
- * * if denotes a file, it's location will be opened and the file selected
+ * - directory -> opn in file browser
+ * - file -> open directory and select file
  *
  * On some platforms the operation may be unsupported. In that case this method is a no-op.
  */
 fun File.browse() = toURI().browse()
 
 /**
  * Browse uri:
- * * if denotes an uri, opens it in its predefined internet browser
- * * if denotes a directory it will be opened
- * * if denotes a file, it's location will be opened and the file selected
+ * - url -> opens it in its standard browser
+ * - directory -> opn in file browser
```

Comment:
Also incorrect. File or dir, its parent is opened and the dir/file selected. We can add that if this behavior is unsupported, the directory (for dir) and parent dir (for file) will be opened instead.